enterprise_access/apps/content_assignments/management/commands/clear_pii_for_expired_assignments.py

```diff
@@ -0,0 +1,78 @@
+"""
+Management command to clear PII from expired assignment records.
+
+This command should be run daily, after the automatic expiration job has completed
+and expiration emails have been sent.
+"""
```

Comment:
It actually doesn't much matter when this job as run, as the logic of your task only affects expired assignment records that have been marked as successfully having sent an expiration email notification.